addons/web_editor/static/src/scss/web_editor.backend.scss

```diff
@@ -72,7 +72,7 @@
         z-index: 1001 !important;
         border: 0;
     }
-    * {
+    > *:not(.modal, .modal-backdrop) {
```

Comment:
@sdegueldre I rebased on saas-13.3 and changed the target and I was about to merge but in fact this is not valid CSS here. `:not(<multiple-selectors>)` is a jQuery syntax, so I am not sure how this achieved the behaviour you desired?

Reply:
@qsm-odoo fixed :+1: